### DISCUSSION TITLE
Sprint 5 meeting | 2022-11-16

### DISCUSSION THREAD

**Discussion by HuangJunye** [QPong/planning#35] [Meetings]
## Agenda

- [x] Sprint review (10 min)
- [x] Sprint retro (10 min)
- [x] Sprint planning (10 min)
- [ ] Q&A (5 min)
- [ ] Game dev resources and news (5 min)

**Comment by HuangJunye** [comment]
## Review

[Last sprint planning](https://github.com/QPong/planning/discussions/34#discussioncomment-4047372)

### Dayeong

### Unathi

### Siddharth

### Junye 

### Follow up

**Reply by HuangJunye** [reply]
2h hacking session (Junye, Dayeong, Unathi)

**Comment by HuangJunye** [comment]
## Retro

### What went well


### What didn't go well


### What to improve?

**Reply by HuangJunye** [reply]
### What went well
- Hacking session is great. We got a lot of things figured out in 2 hours.

### What didn't go well
- We still don't have a working prototype and time is running out. Final showcase happens in 3 weeks.

### What to improve?
- Do another or two hacking sessions together to finish the project.

**Reply by Unathi-Skosana** [reply]
## What went well?

- Hacking session with team
- Successfully grokked the `pico-socket` and its internal workings. Working (but buggy) MVP using the server
- Figured out a solution of how to send fractional and signed numbers via the gpio array during this meeting

## What didn't go well?
- Nothing major. Just gotchas moments while using the gpio array exposed by the `pico-socket` i.e nothing being able to send fractional and signed numbers as one would normally do.

## What to improve?
- Turn over time.

**Reply by Siddharthgolecha** [reply]
What went well

- The feature was implemented. The hacking was great.

What didn't go well

- The feature had some serious bugs which needed to be resolved

What to improve?

- Improve the implemented feature .

**Comment by HuangJunye** [comment]
## Planning

### Dayeong

### Unathi
- Fix fractional and negative number issues
- Drawing paddles with probability
- Room selection
- Deploy the server on the cloud

### Siddharth
- Activate the bomb when the ball hits it
- Only allow one bomb at a time (do not allow new bomb to be added if there is already a bomb)
- Implement all 4 types of bombs and its visual / audio effects

### Junye

**Reply by HuangJunye** [reply]
Another or two hacking sessions
